test/core/surface/completion_queue_test.cc

```diff
@@ -391,9 +405,11 @@ static void test_callback(void) {
   for (size_t pidx = 0; pidx < GPR_ARRAY_SIZE(polling_types); pidx++) {
     int sumtags = 0;
     int counter = 0;
+    gpr_mu_lock(&mu);
+    cb_counter = 0;
```

Comment:
Similar comment to above about whether locks are really needed on this initialization

Reply:
Fixed as above. PTAL